Deploy/stacks/Chile/stack-manager/inputs/data/webspace/index.html

```diff
@@ -0,0 +1,389 @@
+
+</head>
+
+<body lang=EN-GB link=blue vlink="#954F72" style='word-wrap:break-word'>
+
+<div class=WordSection1>
+
+<p class=MsoTitle><img width=1085 height=96 id="Picture 2"
+src="welcome_v8_files/image001.png"></p>
+
+<p class=MsoNormal><span style='font-family:"Verdana",sans-serif'>&nbsp;</span></p>
+
+<p class=MsoTitle><span style='font-size:24.0pt;font-family:"Verdana",sans-serif;
+color:black'>Welcome to Chile's energy grid and energy transition visualisation
+platform </span></p>
+
+<p class=MsoNormal>&nbsp;</p>
+
+<p class=MsoNormal><span style='font-size:24.0pt;line-height:107%'>&nbsp;</span></p>
+
+<p class=MsoNormal><span style='font-size:16.0pt;line-height:107%;font-family:
+"Verdana",sans-serif'>&nbsp;</span></p>
+
+<p class=MsoNormal><span style='font-size:16.0pt;line-height:107%;font-family:
+"Verdana",sans-serif'>Sustainability is one of the most important 21<sup>th</sup>
+century challenges to be addressed. However, given the problem is
+multidisciplinary and requires a multidimensional solution, it is not possible
+to solve this problem without the participation of a broad (if not complete)
+participation of communities. </span></p>
+
+<p class=MsoNormal><span style='font-size:16.0pt;line-height:107%;font-family:
+"Verdana",sans-serif'>&nbsp;</span></p>
+
+<p class=MsoNormal><span style='font-size:16.0pt;line-height:107%;font-family:
+"Verdana",sans-serif'>To address such a complex challenge with massive
+implications, communication skills are key to achieve massive solutions. Many
+efforts have been made in this direction, but not yet enough with up-to-date
+technology. For instance, David Mackay has addressed this problem in his book <b>Sustainable
+energy without the hot air</b> where he emphasizes the magnitude of different
+sorts of energy in an understandable way to the general public. </span></p>
+
+<p class=MsoNormal><span style='font-size:16.0pt;line-height:107%;font-family:
+"Verdana",sans-serif'>&nbsp;</span></p>
+
+<p class=MsoNormal><span style='font-size:16.0pt;line-height:107%;font-family:
+"Verdana",sans-serif'>&nbsp;</span></p>
+
+<p class=MsoNormal><span style='font-size:16.0pt;line-height:107%;font-family:
+"Verdana",sans-serif'>In this visualisation, we have addressed Chile's
+energy grid as a use case, given the role this country might have in the coming
+years as a renewable energy producer. A guided tour is available in the
+following document </span><span style='font-size:16.0pt;line-height:107%'><a
+href="http://theworldavatar.io/demos/chile/guide.pdf"><span
+style='font-family:"Verdana",sans-serif;color:#0B4452'>VISUALISING GUIDE</span></a></span><span
+style='font-size:16.0pt;line-height:107%;font-family:"Verdana",sans-serif'>.
+The following <i>User Targeted Visualisations</i> have been developed:</span></p>
+
+<p class=MsoNormal><span style='font-size:16.0pt;line-height:107%;font-family:
+"Verdana",sans-serif'>&nbsp;</span></p>
+
+<p class=MsoListParagraph style='text-indent:-18.0pt'><b><span
```

Comment:
The names of the visualisations need to match the names that we used for them in the Supplementary Information.

Reply:
done